app/responders/core_responder.rb

```diff
@@ -0,0 +1,11 @@
+class CoreResponder < ApplicationResponder
+    #ALLOWED_TYPES = %w(school department course section).freeze
+
+   topic :course_change
+   topic :section_change
+
+    def respond(event)
+       respond_to :section_change , event.to_json(:only => [:uuid,:course_id, :name, :crn, :seats, :seats, :seats_taken, :created_at, :updated_at, :num_periods, :instructors, :conflicts ])
```

Comment:
So it looks like right here we are writing a message to both the `section_change` and `course_change` topics. We want to only write to the appropriate topic depending on whether it is a course or a section. I think the right way to do this according to the karafka best practices would be to create two separate classes, `CoursesResponder` and `SectionsResponder`, and call them accordingly.